cognite_toolkit/modules/cdf_data_pipeline_asset_default/auth/asset.processing.groups.yaml

```diff
@@ -0,0 +1,39 @@
+# This role is used for the processing of asset data, creating an asset hierarchy 
+name: 'asset:{{location_name}}:processing'
+sourceId: '{{asset_location_processing_group_source_id}}'
+metadata:
+  origin: 'cdf-project-templates'
+  module_version: '{{module_version}}'
+capabilities:
+  - rawAcl:
+      actions:
+        - READ
+        - WRITE
+      scope:
+        tableScope: {
+          dbsToTables: ['src-asset-{{location_name}}-{source_name}']
+        }
+  - transformationsAcl:
+      actions:
+        - READ
+        - WRITE
+      scope:
+        scope:
+          all: {}
+  - sessionsAcl:
+      actions:
+        - READ
+        - CREATE
+        - DELETE
+      scope:
+        all: {}
+  - assetsAcl: 
+      actions:      
+        - READ
+        - WRITE
+      scope: 
+        datasetScope: {
+            ids: ['asset:{{location_name}}']
```

Comment:
I was just reminded that we also have an asset hierarchy scope as well. Not for the processing maybe, where data set is more appropriate, but for read/only consumption. Is this scope being used or should we stay away?